api/proto/teleport/legacy/types/types.proto

```diff
@@ -5700,6 +5700,13 @@ message PluginSpecV1 {
     // Settings for the ServiceNow plugin
     PluginServiceNowSettings serviceNow = 10;
   }
+
+  // generation contains a unique ID that should:
+  // - Be created by the backed on plugin creation.
+  // - Be updated by the backend if the plugin is updated in any way.
+  //
+  // For older plugins, it's possible for this to be empty.
+  string generation = 11;
```

Comment:
Is this also an ordinal, as suggested by the name "generation" or just some random ID? If it is an ordinal, can you add that to the comment. Not sure whether a reader of this field should expect it to increase or just change when the generation changes.

Reply:
I have this as a random ID. Just something random, really.